src/omv/modules/py_tf.c

```diff
@@ -265,41 +265,83 @@ STATIC py_tf_model_obj_t *py_tf_load_alloc(mp_obj_t path_obj) {
     }
 }
 
+typedef enum {
+    PY_TF_SCALE_NONE,
+    PY_TF_SCALE_0_1,
+    PY_TF_SCALE_N1_1,
+    PY_TF_SCALE_N128_127
+} py_tf_input_scale_t;
+
+typedef enum {
+    PY_TF_R_MEAN,
+    PY_TF_G_MEAN,
+    PY_TF_B_MEAN,
+    PY_TF_R_STDDEV,
+    PY_TF_G_STDDEV,
+    PY_TF_B_STDDEV,
+    PY_TF_INPUT_NORMALIZATION_SIZE
+} py_tf_input_normalization_t;
+
 typedef struct py_tf_input_callback_data {
     image_t *img;
     rectangle_t *roi;
+    py_tf_input_scale_t input_scale;
+    float input_normalization[PY_TF_INPUT_NORMALIZATION_SIZE];
```

Comment:
```suggestion
    py_tf_input_scale_t input_scale;
    float input_stats[PY_TF_STATS_SIZE];
```

Reply:
This is an outdated comment. I changed it previously to what was discussed in the meeting with mean/std.